doc/release/upcoming_changes/24940.new_feature.rst

```diff
@@ -0,0 +1,4 @@
+``svdvals`` for `numpy.linalg`
+------------------------------
+
+`numpy.linalg.svdvals` has been added, compatible with Array API.
```

Comment:
Can you also add a sentence explaining what it does?

Reply:
I extended the release note a bit.